src/main/java/eu/europeana/metis/sandbox/service/util/XsltUrlUpdateServiceImpl.java

```diff
@@ -60,7 +66,10 @@ public void updateXslt(String defaultXsltUrl) {
   }
 
   private void saveDefaultXslt(String newTransformXslt) {
+    final Lock lock = lockRegistry.obtain("saveDefaultXslt");
```

Comment:
Comment not blocking this current pr:  

Since we are actually overwriting the value of an existent object in the db, I wonder if we should actually be also locking(in relation to this lock) during the retrieval of that value when a dataset goes through transformation. Perhaps there can be a race condition in that case.
Something to look at in the future.

Reply:
true, if this is being updated, then one node can pick the wrong xlst template and other can pick the correct one.

Reply:
added this ticket to the backlog for following up. https://europeana.atlassian.net/browse/MET-5085